### PR TITLE
Add blue

### LINKED ISSUE #18677: Package request: blue
### Package name

blue

### Package version

Newest

### Package website

[GitHub](https://github.com/grantjenks/blue)
[PyPI](https://pypi.org/project/blue/)

### Package availability

[GitHub](https://github.com/grantjenks/blue)
[PyPI](https://pypi.org/project/blue/)

### Additional comments

No response

### Package is not available

- [X] The package is not available on conda-forge.

### No previous issues or open PRs

- [X] No previous issue exists and no PR has been opened.

### PR DESCRIPTION
**Pull request by BastianZim**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

Closes #18677

### CODE REVIEW

**Comment by BastianZim** [comment]
@conda-forge/help-python Ready for review, thanks!

File: recipes/blue/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+test:
+  imports:
+    - blue
+  commands:
+    - pip check
+    - blue --help
+  requires:
+    - pip
+
+about:
```
Comment (carterbox) [inline recipes/blue/meta.yaml:38]: Please fix this section. The summary is not helpful and the dev_url key has copypasta.

File: recipes/blue/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+    - blue=blue:main
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
+    - pip
+    - python >=3.6
+    - tox
```
Comment (carterbox) [inline recipes/blue/meta.yaml:23]:
```suggestion
```
tox is only required for tests?

**Review by carterbox — CHANGES_REQUESTED** [review]

File: recipes/blue/meta.yaml (hunk 1 above)
Comment (BastianZim) [reply recipes/blue/meta.yaml:37]: Whops, trusted grayskull too much there, thanks for noticing.

File: recipes/blue/meta.yaml (hunk 2 above)
Comment (BastianZim) [reply recipes/blue/meta.yaml]: It is used in the setup.py but let's see if it gets imported or not.

**Comment by BastianZim** [comment]
Thanks!

**Pull request merged**